wp-content/themes/missilethreat/archive.php

```diff
@@ -43,26 +45,137 @@
 
 </header>
 
-<div class='archive'>
+<div class='archive__content'>
 
-  <section class="archive__content">
-    
   <?php
+  if ( is_post_type_archive('missile') ) { ?>
+
+    <section class="actors">
+      <div class="archive__desc">Ballistic missiles, cruise missiles, rockets, artillery, and mortars (RAM), and even maneuvering hypersonic boost glide delivery systems now form the complicated 21st century strike complex with which U.S., allied, and partner nations must contend. Organized by country, the following represents a growing collection of information on global missile systems, with illustrations and up-to-date information on their capabilities and history.</div>
+
+      <?php
+      $args = array(
+        'post_type' => 'actors',
+        'numberposts' => -1,
+        'orderby' => 'meta_value',
+        'order' => 'ASC',
+        'meta_key' => '_actors_archive_name'
+      );
+      $actors = get_posts( $args ); ?>
+
+      <h2 class="actors__header actors__header--lower">Countries</h2>
+        <div class="actors__group actors__group-countries">
```

Comment:
Structurally, these are list of items. Instead of using divs, I would think about reorganizing using an unordered list.